staging/src/k8s.io/client-go/dynamic/fake/client_pool.go

```diff
@@ -21,10 +21,34 @@ package fake
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/testing"
 )
 
+// NewFakeClientPool returns an instance of FakeClientPool which respects manipulations
+// on resources and put them into storage.
+func NewFakeClientPool() *FakeClientPool {
+	o := testing.NewObjectTracker(scheme, codecs.UniversalDecoder())
```

Comment:
For the reviewers, this constructor can help users to use fake `scheme` and fake `codecs`